mxcube3/routes/scutils.py

```diff
@@ -101,10 +107,10 @@ def mount_sample(beamline_setup_hwobj,
 
 def mount_sample_clean_up(sample):
     try:
-        msg = '[SC] mounting %s (%r)', sample['location'], sample['sampleID']
+        msg = '[SC] mounting %s (%r)' % (sample['location'], sample['sampleID'])
         logging.getLogger('HWR').info(msg)
         set_sample_to_be_mounted(sample['sampleID'])
-        set_current_sample(sample['sampleID'])
+        set_current_sample(sample)
         mxcube.CURRENTLY_MOUNTED_SAMPLE_DATA = sample
```

Comment:
Seems like you forgot this one then .... or ?

Reply:
thanks!